workspaces/ui-v2/src/components/ShapeTypeSummary.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { JsonType } from '@useoptic/optic-domain';
+
+import { IShapeRenderer } from '<src>/types';
+import * as Theme from '<src>/styles/theme';
+
+export const ShapeTypeSummary: FC<{
+  shapes: IShapeRenderer[];
+  required: boolean;
+}> = ({ shapes, required }) => {
+  const optionalText = required ? '' : ' (optional)';
+  const components = shapes.map(({ jsonType }: { jsonType: JsonType }) => (
+    <span key={jsonType} style={{ color: Theme.jsonTypeColors[jsonType] }}>
```

Comment:
I'm not sure we want to render type colours of types in the viewer mode. The colour shift is one of the key differences we rely on to clearly differentiate the editor from the viewer, making it appear interactive rather than the same static UI they see in documentation view. It _might_ hold up without that differentiation, but as this is a new interaction paradigm we're introducing, I'd rather differentiate too much rather than too little.

Reply:
Good point, I've reverted / made the color changes only in the edit view